components/service/glean/src/main/java/mozilla/components/service/glean/private/EventMetricType.kt

```diff
@@ -57,26 +56,7 @@ data class EventMetricType<ExtraKeysEnum : Enum<ExtraKeysEnum>>(
         // might get executed.
         val monotonicElapsed = SystemClock.elapsedRealtime()
 
-        // Convert the extra key enums to strings before passing to the storage engine
-        val extraStrings = extra?.let { extra ->
-            // There are two extra "keys" in play here:
-            //   1. The Kotlin enumeration names, in CamelCase
-            //   2. The keys sent in the ping, in snake_case
-            // Here we need to get (2) to send in the ping.
```

Comment:
Can we retain this comment?

Reply:
Oh, sure. I was going to move this to method but lost it somewhere.
Thanks for your observation!